GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2_truss.cfg

```diff
@@ -0,0 +1,207 @@
+
+//	ISS Solar arrays & trusses (P4/S4 - P6/S6)
+//	(P3/S3 truss is a seperate part in HT2 so isn't factored into the solar array config)
+//	Diameter: 10.7m (35ft)
+//	Length: 73.2m (240.2ft)
+//	Mass: 15,900kg //15900kg (#P3/4); 16183kg (#S3/4) split the difference for the mass of the two S3/P3 trusses
+//	Source: https://space.skyrocket.de/doc_sdat/its-s3-4.htm
+//	Source: http://spacecraftearth.com/#smoothscroll-portfolio-module
+@PART[ht2_solarArray_duo]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.81
+	@mass = 15.900
+	@title = ISS P4-P6/S4-S6
+	@manufacturer = Boeing
+	@description = The International Space Station's main source of energy is from three of the four large U.S.-made photovoltaic arrays currently on the station, sometimes referred to as the Solar Array Wings (SAW). The first pair of arrays are attached to the P6 truss segment, which was launched and installed on top of Z1 in late 2000 during STS-97. The P6 segment was relocated to its final position, bolted to the P5 truss segment, in November 2007 during STS-120. The second pair of arrays was launched and installed in September 2006 during STS-115, but they didn't provide electricity until STS-116 in December 2006 when the station got an electrical rewiring. A third pair of arrays was installed during STS-117 in June 2007. A final pair arrived mid March 2009 on STS-119. More solar power was to have been available via the Russian-built Science Power Platform, but it was canceled.
+	@maxTemp = 1073.15
+
+	// Area gives each module 30 kW at tech level 6
+	SolarConfig
+	{
+		area = 113.21
+		level = 6
+	}
+
+	// Disable SolarConfig from changing the dummy module for solar tracking
+	@MODULE[ModuleDeployableSolarPanel]:HAS[#pivotName[SARJ]]
```

Comment:
This remains a bizarre solution from the original mod, with clearly-presented drawbacks.  Can we do this with just a ModuleDeployablePart instead?

Reply:
I tried implementing this, but with no success. If someone else is able to get it to work, that would be great